python/pyspark/resource/profile.py

```diff
@@ -155,12 +155,9 @@ def require(
                 )
         else:
             if self._java_resource_profile_builder is not None:
-                if (
-                    resourceRequest._java_executor_resource_requests is not None  # type: ignore[attr-defined]
-                ):
-                    self._java_resource_profile_builder.require(
-                        resourceRequest._java_executor_resource_requests  # type: ignore[attr-defined]
-                    )
+                r = resourceRequest._java_executor_resource_requests  # type: ignore[attr-defined]
```

Comment:
this is consistent with another place in this file. I think it was just a mistake.

Reply:
Yes, it looks like that.